sdk/storage/AzureStorageBlob/Source/TransferManager.swift

```diff
@@ -63,13 +63,13 @@ internal protocol TransferManager {
 /// A delegate to receive notifications about state changes for all transfers managed by a `StorageBlobClient`.
 public protocol TransferDelegate: AnyObject {
     /// A transfer's state has changed, and progress may be reported.
-    func transfer(_: Transfer, didUpdateWithState: TransferState, andProgress: Float?)
```

Comment:
This provides the recommended internal labels when using XCode's "fix-it" function to stub out protocol methods.